dascore/viz/waterfall.py

```diff
@@ -80,7 +83,7 @@ def waterfall(
     """
     ax = _get_ax(ax)
     cmap = _get_cmap(cmap)
-    data = patch.data
+    data = np.log10(patch.data) if log else patch.data
```

Comment:
Good point! I was considering having the user ensure that their patch has only positive values to avoid getting NaN values. However, it would be better to take the absolute values first.